src/cascadia/PublicTerminalCore/PublicTerminalCore.vcxproj

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Given this is a new C/C++ project, would you mind turning on the AuditMode configuration for it and seeing how many warnings there are (and perhaps fixing them)?

We're working on an effort to slowly turn on AuditMode (which is static analysis) to all C++ projects over time.

We're hoping all new projects can just start out of the gate with static analysis on so they don't build up warning debt over time.

If it's exceptionally onerous to turn on in this PR, please let me know how many warnings there are and we can file a follow up work item to turn it on shortly after this is merged.

Reply:
Oh, I didn't realize I needed to turn that on explicitly. I'll get that turned on asap.

Reply:
Yeah, sorry, it's off by default because to turn it on for our entire solution results in tens of thousands of warnings. So we've been rolling it out project-by-project.

Reply:
I turned it on but it has an issue building I think since some of the libraries that this project depends on don't have auditmode turned on. Does that sound correct or did I just enable it incorrectly?

Reply:
It should only affect the project at hand. I'll look more closely at your SLN change and pull the branch locally to see what I can see.